packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -120,7 +120,7 @@ describe("normalizeAndValidateConfig()", () => {
           - Expected \\"build.cwd\\" to be of type string but got 666.
           - Expected \\"build.watch_dir\\" to be of type string but got 777.
           - Expected \\"legacy_env\\" to be of type boolean but got \\"FOO\\".
-          - Expected \\"main\\" to be a string but got 111
+          - Expected \\"main\\" field to be a string but got 111.
```

Comment:
Let's replace everywhere it says "field to be" to "to be" so these messages are consistent?

Reply:
Yeah - I agree. I didn't do it here because it results in a whole load of test changes that would add noise to this PR.
But since you mention it, I'll do it now.